### PR TITLE
Whats new july

### PR DESCRIPTION
**Pull request by BillWagner**
Switch from our sprint dates to the month start and end dates.

[Internal review link](https://review.docs.microsoft.com/en-us/dotnet/whats-new/dotnet-docs-2020-07?branch=pr-en-us-19801)

### CODE REVIEW

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+title: ".NET docs: What's new for June 28, 2020 - August 1, 2020"
```
Comment (BillWagner) [inline docs/whats-new/dotnet-docs-2020-07.md:2]: I think I should change this to the month:

```suggestion
title: ".NET docs: What's new for July 2020"
```

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+description: "What's new in the .NET docs for June 28, 2020 - August 1, 2020."
+ms.date: 08/03/2020
+---
+
+# .NET docs: What's new for June 28, 2020 - August 1, 2020
```
Comment (BillWagner) [inline docs/whats-new/dotnet-docs-2020-07.md:7]: same change:

```suggestion
# .NET docs: What's new for July 2020
```

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+Welcome to what's new in the .NET docs from June 28, 2020 through August 1, 2020. This article lists some of the major changes to docs during this period.
```
Comment (BillWagner) [inline docs/whats-new/dotnet-docs-2020-07.md:9]: one more:

```suggestion
Welcome to what's new in the .NET docs from July 2020. This article lists some of the major changes to docs during this period.
```

**Review by scottaddie — APPROVED** [review]

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (30)
+- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (10)
+- [imba-tjd](https://github.com/imba-tjd) - 谭九鼎 (2)
+- [LauraKellerGitHub](https://github.com/LauraKellerGitHub) - Laura Keller (2)
+- [tonytins](https://github.com/tonytins) - Anthony Leland (2)
+- [azure-sdk](https://github.com/azure-sdk) - Azure SDK Bot (1)
```
Comment (scottaddie) [inline docs/whats-new/dotnet-docs-2020-07.md:56]: This is a Microsoft-owned service account.

```suggestion
```

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+## Community contributors
+
+The following people contributed to the .NET docs during this period. Thank you! Learn how to contribute by following the links under "Get involved" in the [what's new landing page](index.yml).
+
+- [Youssef1313](https://github.com/Youssef1313) - Youssef Victor (30)
+- [NextTurn](https://github.com/NextTurn) - Next Turn (20)
+- [pkulikov](https://github.com/pkulikov) - Petr Kulikov (10)
+- [imba-tjd](https://github.com/imba-tjd) - 谭九鼎 (2)
+- [LauraKellerGitHub](https://github.com/LauraKellerGitHub) - Laura Keller (2)
```
Comment (scottaddie) [inline docs/whats-new/dotnet-docs-2020-07.md:54]: This is a v-dash account.

```suggestion
```

File: docs/whats-new/dotnet-docs-2020-07.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
+---
```
Comment (Youssef1313) [inline docs/whats-new/dotnet-docs-2020-07.md:1]: Rename to just `2020-07.md`?

Based on @mairaw comment (https://github.com/dotnet/docs/pull/19257/#discussion_r447305779).

**Pull request merged**